routes.go

```diff
@@ -0,0 +1,57 @@
+package main
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	"github.com/gorilla/mux"
+)
+
+type route struct {
+	Name        string
+	Method      string
+	Pattern     string
+	HandlerFunc http.HandlerFunc
+}
+
+var routes = []route{
+	route{
+		"hello",
+		"GET",
+		"/hello/{name}",
+		helloHandler,
+	},
+	route{
+		"initiateTranscriptionJob",
+		"DELETE",
+		"/add_job",
+		initiateTranscriptionJobHandler,
+	},
```

Comment:
don't forget about `{"health", "GET", "/health", healthHandler}`

also, make sure to add `healthHandler`
`func healthHandler(w http.ResponseWriter, r *http.Request) {
 	w.Write([]byte("healthy!"))
 }`